Managers/SettingsMan.cpp

```diff
@@ -44,6 +44,8 @@ namespace RTE {
 		m_SimplifiedCollisionDetection = false;
 		m_SceneBackgroundAutoScaleMode = 1;
 		m_DisableFactionBuyMenuThemes = false;
+		m_PathFinderGridNodeSize = 20;
```

Comment:
Hmm, I'm not sure about this. It's kinda coincidental that it's 20 - it was 20 even when ppm was 24. Setting it to ppm makes it seem like it's a deliberate decision when it's just coincidence.

Reply:
PPM was never 24 in any released build since the dawn of time.
It was shortly 24 because I wanted to see how it plays when ppm matches the standard module 1x1 block size (24x24), and it was reverted shortly after because 4zk said it's kinda ass.

So this coincidence is probably not a coincidence at all.

Reply:
Ahhh whoops, I mixed em up. Fair enough, I guess.